src/main/client/scene/game-scene.ts

```diff
@@ -17,38 +21,108 @@ export class GameScene extends Scene {
       });
    }
 
+   private sceneWidth: number;
+   private sceneWidth2: number;
```

Comment:
why is there is a second screen width

Reply:
oh ok, its half of it

Reply:
Yes we can cean up sum clean-code work later, just wrote a bunch of poc here :D